GiniVision/Classes/CameraViewController.swift

```diff
@@ -256,7 +314,8 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
                 // Retrieve image from default image view to make sure image was set and therefor the correct states were checked before.
                 if let image = self.defaultImageView?.image,
```

Comment:
I'm not sure we need this default debug image anymore. In the simulator, we can use the camera roll to import an image. But I'm also ok with leaving it here.